orchestrator-salt/src/main/resources/salt/salt/top.sls

```diff
@@ -44,6 +44,7 @@ base:
   'G@roles:manager_agent':
     - cloudera.repo
     - cloudera.agent
+    - datalake_metrics.get_solr_hbase_data_sizes
```

Comment:
do we need this at the moment? We can run states from java code and the command line when need it, I think we can remove this now. Same for: 'roles:postgresql_server':

Reply:
Okay so this is something I'm not too familiar with, I assumed that specifying the state here just ensured that it and its necessary files would be present on the nodes associated with these roles. Is that not correct?
If it is, wouldn't removing this make it not runnable on those nodes?

Sorry if I'm confused on this my Salt knowledge is limited.

Reply:
We will upload the whole salt directories: https://github.com/hortonworks/cloudbreak/blob/53fa7b7bc92d57c91165d5bb78b618f070e321d2/orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/SaltOrchestrator.java#L1104-L1107
(these directories: https://github.com/hortonworks/cloudbreak/tree/53fa7b7bc92d57c91165d5bb78b618f070e321d2/orchestrator-salt/src/main/resources)

So the sls files will be there even if you not define it in top.sls